src/launcher/java/org/truffleruby/launcher/options/CommandLineOptions.java

```diff
@@ -54,6 +53,10 @@
     private boolean forceStdin;
     private boolean shouldPrintShortUsage;
 
+    // TODO (pitr-ch 26-Jul-2017): Move to Options when implementing -s option
+    // Currently not used
```

Comment:
No there is also the parsing logic in the processArgv methods, but I've already implemented it so it does not stay behind.